src/game/shipsArray.js

```diff
@@ -0,0 +1,190 @@
+let PLAYER_SHIPS = {
```

Comment:
Better solve the problem in such way than not solving It at all. You can read on Discord another way to solve that problem which I suggest implementing for encapsulation, granulation, and single responsibility principle.